src/index.html

```diff
@@ -7,8 +7,50 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    > -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+
+      <div class="logo">
+        <a href="#home" class="logo__link">
+          <img
+            src="images/logo.png"
+            alt="moyo_logo"
+            class="logo__image"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link is-active" href="#">apple</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">smartphones</a></li>
+          <li class="nav__item">
+            <a
+            data-qa="hover"
+            class="nav__link"
+            href="#">laptops & computers
+            </a>
```

Comment:
Pay attention on indents
```suggestion
            <a
              data-qa="hover"
              class="nav__link"
              href="#">laptops & computers
            </a>
```